jujugui/templates/index.html.mako

```diff
@@ -67,7 +67,7 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
       <div class="header-banner header-banner--left">
         <ul id="browser-nav" class="header-banner__list">
           <li id="nav-brand-env" class="header-banner__list-item">
-            <a class="header-banner__link" href="/">
+            <a class="header-banner__link" href="${logo_url}">
```

Comment:
If this is navigating away it should use `target="_blank"` because in sandbox mode there is no persistence and it would be quite annoying to lose all your state because you clicked this link.